reed-solomon-novelpoly/src/field/inc_log_mul.rs

```diff
@@ -121,3 +238,30 @@ fn cantor_basis() {
         assert_eq!(a, square ^ b);
     }
 }
+
+
+#[cfg(target_feature="+avx2")]
```

Comment:
incorrect target feature detection?
```suggestion
#[cfg(target_feature="avx2")]
```